packages/cli/src/commands/dataMigrate/install.js

```diff
@@ -87,5 +87,6 @@ export const handler = async () => {
     await tasks.run()
   } catch (e) {
     console.log(c.error(e.message))
```

Comment:
Use stderr, instead of stdout
```suggestion
    console.error(c.error(e.message))
```